lib/content/getting-started.md

```diff
@@ -447,6 +443,7 @@ The main takeaway from here is that, out of the box, we get an in-memory databas
 >
 > In fact, there's already a sqlite database, prepopulated with a handful of riddles, [available in the example application repo](https://github.com/hapipal/examples/blob/master/paldo-riddles/riddles.db). As an exercise for the reader, try setting `filename` with an environment variable (as would usually be done in a production deployment (and how the examples repo is setup))
 >
+> NOTE: using the prepopulated database may error because the migration scripts likely don't exist.
```

Comment:
This is a good point: the migration directory will seem corrupt to knex when the user hasn't simply cloned the example repo, since it's expected the exact same migrations (i.e. by their timestamped filename) will exist.  Something to think on, hmm!  CC @zemccartney in case you have any ideas.

Reply:
I pulled off an issue to track this so that we can land the rest of these updates: https://github.com/hapipal/hapipal.com/issues/60

Reply:
Oof, bummer. Sorry, y'all, thanks for catching @openam! Immediate idea is doing away with the prepopulated db and setting up some sort of seeding script — maybe an `onPreStart` extension? — in the example repo. So we could point people to that extension from the getting started tutorial to copy into their project if they so choose, assuming they were creating the example app manually vs. cloning the example repo. People might find it nice that they can see and edit the riddles content, too. I dunno, I'll mull it over a bit more

Reply:
Could you do the prepopulated, but delete the knex tables? That way it will start properly. It would probably cause issues when people start adding migration scripts for the existing tables. 

It could be said that if you don't want to go through the migration stuff here is a prepopulated table...